web/src/app/modules/views/main/editors/modules/tool-pallette/tools/ceg/ceg-layout-tool.ts

```diff
@@ -172,21 +174,32 @@ export class CEGLayoutTool extends ToolBase {
         let layerPositionY = Config.CEG_LAYOUT_CLEARANCE_Y;
         let gridSpace = Config.GRAPHICAL_EDITOR_GRID_SPACE;
         let maxHeight = Math.max(...dimTable.map(d => d.height));
+        let maxWidth = Math.max(...dimTable.map(d => d.width));
 
+        console.log('Inside CEG updateLayout...');
```

Comment:
This should not be here

Reply:
Have removed all debugger statements and console logs in the latest commit.